### ISSUE TITLE
vttcue

### ISSUE THREAD

**Issue by iptvsmartws** [shaka-project/shaka-player#6522]
can any one help , i have a webm video with subtitles text track included in video 

what is the fonction to parse and show it while playing the video ?

**Comment by joeyparrish** [comment]
I don't believe the browser is capable of extracting those subtitles that are embedded in a webm video.  Shaka Player isn't capable of this either, since Shaka Player is designed mostly for formats like DASH and HLS where the subtitles are separate from the video and audio streams.

**Comment by iptvsmartws** [comment]
I consider use the player in smart tv web based app , so more powerfull. Not standard browser limitation

**Comment by joeyparrish** [comment]
And yet smart TVs are all using ports of standard desktop browser engines like Chromium (Chrome, Edge) and WebKit (Safari).  In our experience, they are typically **more** limited than desktops, not less.  They have less memory, their hardware decoders are more picky, etc.  But the parsers and web APIs are the same.

If you know your particular smart TV can handle this scenario, then please proceed.  Sorry we couldn't help you.
